packages/x-tree-view/src/RichTreeView/RichTreeView.tsx

```diff
@@ -146,6 +146,14 @@ RichTreeView.propTypes = {
   // | These PropTypes are generated from the TypeScript type definitions |
   // | To update them edit the TypeScript types and run "yarn proptypes"  |
   // ----------------------------------------------------------------------
+  /**
+   * The ref object that allows Tree View manipulation. Can be instantiated with `useTreeViewApiRef()`.
+   */
+  apiRef: PropTypes.shape({
```

Comment:
We will probably remove the depth 2 proptypes generation later, but for now I think it's good to keep it since it's very short